index.js

```diff
@@ -10,18 +10,18 @@ mongoose.connect(
   'mongodb+srv://BaSD:BaSD2021@cluster0.5vk6q.mongodb.net/mindSet?retryWrites=true&w=majority',
   (error) => {
     if (error) {
-      console.log('Error: ', error)
+      console.log('🔴 Database error: ', error);
     } else {
-      console.log('Database connected')
+      console.log('🟢 Database connected');
```

Comment:
add `// eslint-disable-next-line no-console` before console.log to avoid Eslint warning messages. On both console.logs